src/main/java/org/jabref/gui/mergeentries/newmergedialog/cell/sidebuttons/WarningButton.java

```diff
@@ -0,0 +1,50 @@
+package org.jabref.gui.mergeentries.newmergedialog.cell.sidebuttons;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.scene.control.Button;
+
+import org.jabref.gui.Globals;
+import org.jabref.gui.actions.Action;
+import org.jabref.gui.actions.ActionFactory;
+import org.jabref.gui.actions.SimpleCommand;
+import org.jabref.gui.icon.IconTheme;
+
+import com.tobiasdiez.easybind.EasyBind;
+
+public class WarningButton extends Button {
```

Comment:
Maybe, this should be an information only. Is it possible to show an information sign (like an i in a circle).

Reason: Warning indicates something a user should be carefully looking at and fixing something. In the concrete case, the user does not need to do an action, just decide which "rendering" he prefers more.